examples/vue/web-test-runner.config.js

```diff
@@ -3,4 +3,19 @@ module.exports = {
   plugins: [
     vite(),
   ],
+  coverageConfig: {
+    include: ['src/**']
```

Comment:
Errors will be thrown if `@web/test-runner` tries to generate coverage report for files under `node_modules/.vite/`.
So here we use a whitelist to only include the source files.